src/main/java/com/linkedin/kmf/services/CommitAvailabilityMetrics.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import java.util.Map;
+import org.apache.kafka.common.MetricName;
+import org.apache.kafka.common.metrics.Metrics;
+import org.apache.kafka.common.metrics.Sensor;
+import org.apache.kafka.common.metrics.stats.Rate;
+import org.apache.kafka.common.metrics.stats.Total;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+class CommitAvailabilityMetrics {
+
+  private static final String METRIC_GROUP_NAME = "commit-availability-service";
+  public final Sensor _offsetsCommitted;
+  public final Sensor _failedCommitOffsets;
+  private static final Logger LOG = LoggerFactory.getLogger(CommitAvailabilityMetrics.class);
+
+  /**
+   * Metrics for Calculating the offset commit availability of a consumer.
+   * @param metrics the commit offset metrics
+   * @param tags the tags associated, i.e) kmf.services:name=single-cluster-monitor
+   */
+  CommitAvailabilityMetrics(final Metrics metrics, final Map<String, String> tags) {
+    LOG.info("{} called.", this.getClass().getSimpleName());
+    _offsetsCommitted = metrics.sensor("offsets-committed");
```

Comment:
I'm not sure how quantile metrics are computed with the metric library being used, but perhaps there should be  some kind of 99th percentile commit time.  Is this the CommitLatencyService?

Reply:
The class in discussion is for the `CommitAvailabilityMetrics`. The `CommitLatencyService` will be sampling and determining the 99th percentile commit time for its calculation.